### PR TITLE
🐛 corret merge stats' start and end date

### PR DESCRIPTION
**Pull request by stormslowly**
as the title

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -54,14 +54,41 @@ const getAllTests = flatMap(suite => [
...
+    (currentMaxSpan, span) => {
+      const start = new Date(
+        Math.min(currentMaxSpan.start.getTime(), span.start.getTime())
+      )
+      const end = new Date(
+        Math.max(currentMaxSpan.end.getTime(), span.end.getTime())
+      )
+      return { start, end }
+    },
+    { ...spans[0] }
```
Comment (Antontelesh) [inline lib/index.js]: Why do you need a spread here? I think, here it’s possible to reduce without supplying the initial value at all

**Comment by rohitkadam19** [comment]
@Antontelesh @stormslowly  any updates here? Would love to get this merged. I am trying to generate dashboard and correct values in stats after merge will be helpful :). Thanks

File: lib/index.js (hunk 1 above)
Comment (stormslowly) [reply lib/index.js:72]: yes, it can be removed

**Review by Antontelesh — APPROVED** [review]

**Comment by stormslowly** [comment]
@Antontelesh  thx a lot for your comment,  learn the reduce API again ~  
<3

**Pull request merged**